GenericDiffractometer.py

```diff
@@ -185,6 +208,10 @@ def __init__(self, name):
              GenericDiffractometer.CENTRING_METHOD_MOVE_TO_BEAM: \
                  self.start_move_to_beam}
 
+        self.connect(self, 'equipmentReady', self.equipment_ready)
```

Comment:
Hi @JieNanMAXIV  and other

I accepted this PR without testing, but already now see a issue.
equipment_ready is not defined in GenericDiffractometer (it is in MAXIV/BIOMAXMiniDiff.py)
Should we add this method to HardwareObject in HardwareRepository?